projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -98,8 +99,19 @@ public static function enqueue_scripts() {
 		wp_add_inline_script( 'my_jetpack_main_app', Connection_Initial_State::render(), 'before' );
 	}
 
+	/**
+	 * Product data
+	 *
+	 * @return array Jetpack products on the site and their availability.
+	 */
+	public static function get_products() {
+		return array();
+	}
+
 	/**
 	 * Echos the admin page content.
+	/**
+	 * Echoes the admin page content.
```

Comment:
it seems to be kind of duplicated?

Reply:
fixed in 2e6e56610fe3dea0abb85ebce28d1a792b917a51